src/Type/Constant/ConstantArrayType.php

```diff
@@ -314,7 +315,7 @@ public function equals(Type $type): bool
 
 	public function isCallable(): TrinaryLogic
 	{
-		$typeAndMethod = $this->findTypeAndMethodName();
+		$typeAndMethod = $this->findTypeAndMethodName(new OutOfClassScope());
```

Comment:
Passing out of class scope here is causing grief for CallStaticMethodsRuleTest::testBug1971